graalpy/graalpy-freeze-dependencies-guide/README.md

```diff
@@ -77,25 +86,66 @@ You can use the GraalPy plugin to manage Python packages for you.
 </build>
 ```
 
+`build.gradle.kts`
+```kotlin
+plugins {
+    application
+    id("org.graalvm.python") version "24.1.1"
+}
+```
+
+`build.gradle.kts`
+```kotlin
+graalPy {
+    packages = setOf("vaderSentiment==3.3.2") // ①
+}
+```
+
 ❶ The `packages` section lists all Python packages optionally with [requirement specifiers](https://pip.pypa.io/en/stable/reference/requirement-specifiers/).
 In this case, we install the `vaderSentiment` package and pin it to version `3.3.2`. Because we are not
-specifying `<pythonResourcesDirectory>` the Maven plugin will embed the packages into the
+specifying `<pythonResourcesDirectory>` the plugins will embed the packages into the
 resulting JAR as a standard Java resource.
 
 ## 5. Determining and Pinning Transitive Dependencies
 
-Package the application using Maven, this should install all the transitive dependencies
+When you package the application, it installs all the transitive dependencies
 in a newly created [virtual environment](https://docs.python.org/3/library/venv.html):
 
 ```shell
-mvn package
+./mvnw package
 ```
 
-If the compilation is successful, one can run the following command to get
-versions of all the installed Python packages:
+```shell
+./gradlew build
+```
 
+If the compilation is successful, one can run the following command to get versions of all the installed Python packages if you use Maven:
+
+On macOS and Linux:
 ```shell
 ./target/classes/org.graalvm.python.vfs/venv/bin/pip3 freeze -l
```

Comment:
For the apps generated by `gradle init ...` this would be `./app/target/......`. 

Not sure what to do about this. We want to share the directory structure for both Maven and Gradle for the "reference solution", at the same time should the guide show paths that are applicable to the reference solution or that are applicable to the solution that the user gets when following the guide step by step?

Reply:
Ah, I meant to add this comment to the Gradle section...

Reply:
We could use

```
layout.buildDirectory.set(layout.projectDirectory.dir("app/build"))
```

in the "reference solution" to make the paths the same for both. I've tested that and added it to the branch with suggestions